### ISSUE TITLE
notcurses-info doesn't scroll enough when started close to the bottom

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#1994]
`notcurses-info` works properly when started anywhere in the top portion of the terminal, or at the very bottom. there's a range at the bottom, however, where it doesn't scroll enough, and the capabilities section ends up covering the initial banner. this is presumably a flaw within the CLI model, not `notcurses-info` itself. see screenshot:

![2021-07-27-032343_1047x1417_scrot](https://user-images.githubusercontent.com/143473/127113112-4b505fb5-aabc-4445-8662-0421b7003bd5.png)

**Comment by dankamongmen** [comment]
so i launched one with 11 rows left below the launch line . there are 20 lines of output when stderr is redirected. this setup scrolls 5 too few lines (15 lines are used), and `rasterize_scrolls()` reports an order-10 scroll.

so we're ending up at bottom - 15, and want to be bottom - 20.

we had 11 lines free. 10 lines of scrolling ought have provided us with the 20 necessary lines. why are there only 15?

ahhhh, because we're not scrolling when we need to, but from where we started! that's definitely it. so what if we went to the bottom of the screen, scrolled, and returned to the expected first location? i think that's the thing to do, ought work.
